src/id.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DQLITE_ID_H_
+#define DQLITE_ID_H_
+
+#include <stdint.h>
+
+struct id_state
+{
+	uint64_t data[4];
+};
+
+
+uint64_t idNext(struct id_state *state);
```

Comment:
Maybe add some (short) comments on what these functions are supposed to do / used for.